### DISCUSSION TITLE
[A-L] (2024/25) Foglio 1 - Esercizio 1

### DISCUSSION THREAD

**Discussion by GiusTMP** [sapienzastudentsnetwork/algebra#462] [Esercizi - Viaggi/Cherubini]
![1](https://github.com/user-attachments/assets/9a0628ea-5929-4834-ae71-a15d79622bb4)

**Comment by Rurik-D** [comment]
![image](https://github.com/user-attachments/assets/439326e9-3e65-4357-aae6-3a51c1f8d763)

**Comment by niccolo-fato** [comment]
![photo_2024-09-29_17-22-23](https://github.com/user-attachments/assets/49a1fffa-8b5f-4900-82a4-a91104247f5a)
